### DISCUSSION TITLE
Welcome to core Discussions!

### DISCUSSION THREAD

**Discussion by opoplawski** [pfsensible/core#40] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by philippenc** [comment]
hello, pfsense-this has just been upgraded to 2.7.0-RELEASE and the python version has upgraded to 3.11. Do you know when pfsensible will be compatible with the version of python 3.11? THANKS

**Comment by opoplawski** [comment]
Please start a new discussion for your support question.  But ansible should auto-discover python 3.11.
